src/parsing/index.ts

```diff
@@ -136,3 +141,15 @@ export function parsePostgreSqlWithoutCursor(query: string): ParseResult {
     // In order to truly complete a finished query, we need to add space to it like so "SELECT * FROM |".
     return parsePostgreSql(query + ' ', '');
 }
+
+
+export function parseClickhouse(queryBeforeCursor: string, queryAfterCursor: string): ParseResult {
```

Comment:
In our websql we are writing databases in camel case `ClickHouse`. See the function above `PostgreSql`, not `Postgresql`. Let's make it consistent here also.